src/book/02-system/01-foundations/07-director.mdx

```diff
@@ -324,8 +324,11 @@ Some examples are
 ### Actuation
 
 The actuation layer is where the (real or simulated) hardware is controlled.
-They live in `modules/actuation` in the source code.
-The end of the actuation layer involves emitting servo commands to the HardwareIO or Webots modules.
-
-- ScriptRunner: Executes scripts given to it
-- InverseKinematics: Takes limb positions/head positions and converts them into servo positions
+These modules live in `modules/actuation` in the source code.
+The end of the actuation layer involves emitting servo commands to be picked up by a platform module, such as HardwareIO or Webots modules.
+Modules in this layer include
+
+- Kinematics: Takes end effector positions and converts them into servo positions
+- Sequences: Takes a sequence of servo positions and emits them in order
+- Groups: Takes a group of servos and emits them together, will only emit Done when all servos are done
+- Servos: Takes a single servo and emits it, will only emit Done when the servo has finished moving
```

Comment:
Only when finished moving was an extension you wanted to add. Currently, it'll emit done when out of time.

Reply:
This are just examples. I'm not sure if even having this list here is very good since it should be on the relevant subsystem page, and two places to have to update makes it more likely it'll get out of date.